setup.md

```diff
@@ -1,84 +1,218 @@
 ## Test Setup
 
-This file contains settings needed to setup local testing of odbc. It is most
-useful for someone trying to develop the R package, so they can run the unit
-tests locally.
+This file is intended to help developers of the R package install needed dependencies.
 
-## PostgreSQL and MySQL
+While the odbc package contains some documentation on how to install and configure database drivers in `vignette("setup")`, the documentation assumes that users are connecting to databases that have already been set up. In order to test package functionality, though, odbc sets up small example database deployments.
 
-Run them locally with `brew services start mysql` `brew services start postgres`
-Create a 'test' database and a 'postgres' user with 'password' as the password.
+## RODBC
 
-See the configuration files in .github/odbc for examples.
+We need to install the RODBC package for benchmarking in the vignette `vignette("benchmarks")`. The CRAN version of RODBC uses iODBC, so to use unixODBC we need to recompile it from source, specifying the odbc manager explicitly:
+
+```r
+install.packages("RODBC", type = "source", INSTALL_opts="--configure-args='--with-odbc-manager=odbc'")
+```
+
```

Comment:
Do we need a SQLite entry?

Reply:
Sure thing—I'll tack that on to this PR in a bit.

Reply:
Minimal, but [f79d505](https://github.com/r-dbi/odbc/pull/670/commits/f79d505fe64a8cb668532a084c7349b362cb2ef9).